packages/file_selector/file_selector_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Updates minimum supported SDK version to Flutter 3.10/Dart 3.0.
+
```

Comment:
This one has already been migrated [here](https://github.com/flutter/packages/commit/42dbb7573aa2f8263483f905114dbfaab1810b46#diff-6127965c509bfd98a9a6f317d0dd256ce511d6257aa4912ddee1726230ec0392) (to move from dart:html to package:web), I think this changelog may be reverted.